pkg/statsd/parser.go

```diff
@@ -110,9 +110,13 @@ func (dp *DatagramParser) Run(ctx context.Context) {
 				accumE += eventCount
 				accumB += badLineCount
 			}
+			// TODO: Refactor this to use a MetricConsolidator
```

Comment:
:stare: is there an issue?

Reply:
Yep, #210 

The entirety of this PR was so that when I do #210 it's a small and easy change.